build-tools/packages/build-cli/src/commands/list.ts

```diff
@@ -110,3 +100,31 @@ export default class ListCommand extends BaseCommand<typeof ListCommand> {
 		return filtered;
 	}
 }
+
+/**
+ * Calculates the packages that should be published to a feed and returns a map of Feed to the packages that should be
+ * published there.
+ */
+export function FeedsForPackages(
```

Comment:
This is only used in a test... probably should jsut delete?

Reply:
No strong feelings about it 🤷